pkg/flags/flags.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2019 The Knative Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+)
+
+const (
+	kubeConfig = "kubeconfig"
+	namespace  = "namespace"
+)
+
+// AddTektonOptions amends command to add flags required to initialise a cli.Param
+func AddTektonOptions(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringP(
+		kubeConfig, "k", "",
```

Comment:
Can we have the long form option?

Reply:
we do, don't we? `--kubeconfig `

Reply:
```
Flags:
  -h, --help                help for pipeline
  -k, --kubeconfig string   kubectl config file (default: $HOME/.kube/config)
  -n, --namespace string    namespace to use (mandatory)

```

Reply:
ah nice, i misteaded from the documentation